src/FilamentBreezyServiceProvider.php

```diff
@@ -61,8 +65,22 @@ public function packageBooted(): void
             });
         }
 
-        ResetPassword::createUrlUsing(function ($user, string $token) {
-            return route('filament.password.reset', ['token' => $token]);
+        ResetPassword::createUrlUsing(function ($notifiable, string $token) {
```

Comment:
but I guess there is something wrong here.
this overrides all `ResetPassword` emails.
not only filament emails.

I will test more once I got some time.
but for now this fix should be applied.